### ISSUE TITLE
Woocommerce Product Price Dynamic Data no decimal places for round numbers

### ISSUE THREAD

**Issue by ollycowan** [soflyy/breakdance-bugs#667]
When creating a Global Block for Woo products and adding Product Price through dynamic data, round numbers do not display decimal places. 

e.g. 4.00 displays as 4.

<img width="361" alt="image" src="https://user-images.githubusercontent.com/14874268/213441515-f349bea7-6f44-4d21-8334-d8ccf1a8b73c.png">

SEE >>> https://www.loom.com/share/d29ace76f7ad4d24989b5251b628a231

However, decimal places are set to 2 in Woocommerce settings and using default Breakdance Woo Product List / Builder the price is displayed correctly

<img width="431" alt="image" src="https://user-images.githubusercontent.com/14874268/213441488-a4cbaf05-a710-41b9-9ce0-6a869a6690b9.png">

**Comment by a-dubiel** [comment]
> Woo Product List / Builder the price is displayed correctly

Hey @ollycowan! You should use Woo elements instead because they are fully integrated with Woo data and shop settings. We are planning on shipping more Woo advanced features soon. I assume the only reason you used the global block is to have product title and price aligned horizontally?

**Comment by ollycowan** [comment]
No, I am using a global block / loop builder to create a custom image
layout for the product image.

**Comment by a-dubiel** [comment]
> No, I am using a global block / loop builder to create a custom image

Gotcha. Yeah, we are currently working on this feature. This should be shipped very soon. I will make a note internally to get back to you in this thread. Thanks.

**Comment by ollycowan** [comment]
Cheers Andrzej

**Comment by ollycowan** [comment]
I just realised if i add the decimal places in the product page itself then
these are then displayed in the global block, so that's a workaround for
now.

**Comment by louisreingold** [comment]
Maybe we should add ability to process dynamic data return values with PHP, i.e. `number_format`, etc.
